index.js

```diff
@@ -58,5 +58,7 @@ const giveRatio = () => {
   }
 };
 
+document.querySelector("#copyright").innerText = new Date().getFullYear();
```

Comment:
This looks good, tho I have little experience with DOM. Whole thing looks good in the browser also.
Question: maybe we should place these line related to footer below all stuff related to contrast ratio, not in the middle of it?

Reply:
It doesn't hurt to place it there, but organizationally it would be nice to place it at the bottom